src/modules/selects/SelectPlugin/SelectPlugin.scss

```diff
@@ -4,4 +4,11 @@
     .Select-option.is-focused {
         background-color: $hf-color-snot-green;
     }
+    .Select-value {
+        font-size: $hf-font-size-base;
+        line-height: $hf-height-input !important;
+    }
+    .Select-menu-outer {
+        font-size: $hf-font-size-base;
+    }
 }
```

Comment:
What I would suggest is something like this:

```scss
.j-select {
    font-size: $hf-font-size-base;

    .Select-option.is-focused {
        background-color: $hf-color-snot-green;
    }
    .Select-control {
        .Select-value {
            line-height: $hf-height-input;
        }
    }
}
```

And then also add `font-size: $hf-font-size-base;` rule to `.j-tags-select` and `.j-multiselect` classes